portal/factories/app.py

```diff
@@ -260,8 +260,13 @@ def configure_logging(app):  # pragma: no cover
     app.logger.setLevel(level)
 
     # Configure Error Emails for high level log messages, only in prod mode
+    # and with email enabled
+
     if not any((
-            app.debug, app.testing, not app.config.get('ERROR_SENDTO_EMAIL'))):
+            app.debug,
+            app.testing,
+            not app.config.get('ERROR_SENDTO_EMAIL'),
+            app.config.get('MAIL_SUPPRESS_SEND'))):
```

Comment:
I think this check will disable exception emails if `MAIL_SUPPRESS_SEND` is enabled, which might be useful in the staging environment. 

Not a huge deal either way if we're going to work towards generalizing exception emails with ELK.